doc/FAQ.md

```diff
@@ -9,11 +9,11 @@ User
 * **[How can I upload images, files, links, videos and sound files to posts?](help/FAQ#upload)**
 * **[Is it possible to have different avatars per profile?](help/FAQ#avatars)**
 * **[How can I view Friendica in a certain language?](help/FAQ#language)**
-* **[What is the difference between blocked|ignored|archived|hidden contacts?](help/FAQ#contacts)**
+* **[How do blocked, ignored, archived and hidden contacts behave?](help/FAQ#contacts)**
 * **[What happens when an account is removed? Is it truly deleted?](help/FAQ#removed)**
 * **[Can I subscribe to a hashtag?](help/FAQ#hashtag)**
 * **[How to create a RSS feed of the stream?](help/FAQ#rss)**
-* **[Are there any clients for friendica I can use?](help/FAQ#clients)**
+* **[List of friendica clients](help/FAQ#clients)**
```

Comment:
Why did you change this from a question to a headline even though it's the style we've been using for the FAQ?

Reply:
Thought it to be more compact and quicker to grasp by users. If you want to stick with the question format, we can revert the client list title.